core/applicationstate.cpp

```diff
@@ -0,0 +1,14 @@
+#include "applicationstate.h"
```

Comment:
we like having an SPDX comment on the top of all files

Reply:
Done.